scripts/coder-dev.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# This is a shim for developing and dogfooding Coder so that we don't
+# overwrite an existing session in ~/.config/coderv2
+set -euo pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+# shellcheck disable=SC1091
+source "${SCRIPT_DIR}/lib.sh"
+PROJECT_ROOT=$(cd "$SCRIPT_DIR" && git rev-parse --show-toplevel)
+
+CODER_DEV_DIR="$PROJECT_ROOT/.coderv2/"
+CODER_DEV_BIN="${CODER_DEV_DIR}/coder"
+if [[ ! -d "${CODER_DEV_DIR}" ]]; then
+	mkdir -p "${CODER_DEV_DIR}"
+fi
+
+if [[ ! -x "${CODER_DEV_BIN}" ]]; then
```

Comment:
Idea: Should we try to detect outdated binaries (e.g. long running `develop.sh` scripts?). Something like `if ! coder version | grep -q $(git rev-parse HEAD)`, then print warning?

Reply:
That sounds fun!